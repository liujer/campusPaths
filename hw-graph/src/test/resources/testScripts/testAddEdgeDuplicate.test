# Tests adding of edges by attempting to add
# multiple edges
CreateGraph F
AddNode F one
AddNode F two
AddEdge F one two e12
AddEdge F one two e12
ListNodes F
ListChildren F one
ListChildren F two