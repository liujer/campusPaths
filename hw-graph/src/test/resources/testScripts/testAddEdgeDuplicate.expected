# Tests adding of edges by attempting to add
# multiple edges
Created graph F
Added node one to F
Added node two to F
Added edge e12 to F
Could not add edge e12 to F
ListNodes of F output: one two
ListChildren of one in F output: two
ListChildren of two in F output: